python/kfserving/docs/KFServingClient.md

```diff
@@ -19,6 +19,7 @@ Class | Method |  Description
 KFServingClient | [set_credentials](#set_credentials) | Set Credentials|
 KFServingClient | [create](#create) | Create KFService|
 KFServingClient | [get](#get)    | Get the specified KFService|
+KFServingClient | [watch](#watch)    | Watch the specified KFService or all KFServices in specified namespace|
```

Comment:
not sure if we need a new API here, we can add an option `watch=sync` to the `get` API to wait for the service to be ready before returning back the `KFService` object imo.

Reply:
@yuzisun Great suggestion! If so, I think watching 10 minutes is better than 15 minutes(current implement). And I think we can also add one option for `get` API so that user can watch the kfservice. Personally we can let type of `watch ` bool (True or False) that's consistent with some k8s python APIs. Thanks!